routing_common/routing_common_tests/vehicle_model_test.cpp

```diff
@@ -363,12 +362,10 @@ UNIT_CLASS_TEST(VehicleModelTest, BicycleModel_Speeds)
     {path, yesBicycle}, // TODO: unpaved by default, so should be lower than shared footways or cycleways, but is equal
     {cycleway, pavedBad},
     {footway, yesBicycle, pavedBad},
+    {footway}, // If allowed in the region.
     {cycleway, unpavedBad},
+    {path, unpavedGood}, // Its controversial what is preferrable: a good path or a bad cycleway
     {path, yesBicycle, unpavedBad},
-    {path, unpavedGood}, // TODO: should be faster than bad surface cycleway (mtb?), but weight is less while eta is faster
-    {footway},
-    {footway, c.GetTypeByPath({"hwtag", "nobicycle"})}, // TODO: should be lower than previous, but is equal
```

Comment:
Why did you remove case with {"hwtag", "nobicycle"} ?

Reply:
Its a bit controversial.
The test is for `AllLimitsInstance` which means that cycling is allowed on footways.
The speed calculation doesn't take the `hwtag-nobicycle` into account (I guess its processed elsewhere and just forbids bici routing there), hence a `nobicycle footway` would appear in this test with the same speed as a regular `footway`, which doesn't make sense.

Though logically an explicit `bicycle=no` should result in a dismount speed:
- https://github.com/organicmaps/organicmaps/issues/9784#issuecomment-2508229612

This PR doesn't change the current behavior, but it increased the footway speed for countries where its allowed.

Reply:
I'll add a todo comment to the test.